### PR TITLE
unwanted files removed

### PR DESCRIPTION
**Pull request by nasrat-codes**
# Unwanted files

- Removed files at #2

### CODE REVIEW

File: extensions.txt [hunk 1]
```diff
@@ -1,28 +0,0 @@
-01: 'Auto Close Tag'
-02: 'Auto Rename Tag'
-03: 'Babel ES6/ES7'
-04: 'Bracket Pair Colorizer'
```
Comment (noor-codes) [inline extensions.txt:4]: These are cool extensions

Comment (noor-codes) [reply extensions.txt:4]: I'll miss them

**Review by noor-codes — APPROVED** [review]
Looks good. ready to merge

File: extensions.txt (hunk 1 above)
Comment (nasrat-codes) [reply extensions.txt:4]: yeah

**Pull request merged**